client/client.go

```diff
@@ -23,10 +30,15 @@ type AuthenticatedCredential struct {
 	ConsumerId string `json:"consumer_id"`
 }
 
+// Called by the plugin server at initialization.
 func New(ch chan interface{}) Client {
 	return Client{bridge.New(ch)}
 }
 
+// kong.Client.GetIp() returns the remote address of the client making the request.
```

Comment:
Should we also include the phases information, as done in the Lua PDK? It's quite important, as otherwise, if used in phases where not supported, it will result in an error. This applies to all other methods, but I would also say it's not a blocker to merge this, in the interest of incremental improvement.